openmc/lattice.py

```diff
@@ -1207,20 +1207,20 @@ def get_local_coordinates(self, point, idx):
 
         """
         if self._orientation == 'x':
-            x = point[0] - (self.center[0] + self.pitch[0]*idx[0] +
-                            0.5*self.pitch[0]*idx[1])
+            x = point[0] - (self.center[0] + self.pitch[0] * idx[0] +
+                            0.5 * self.pitch[0] * idx[1])
             y = point[1] - (self.center[1] +
-                            sqrt(0.75)*self.pitch[0]*idx[1])
+                            sqrt(0.75) * self.pitch[0] * idx[1])
         else:
             x = point[0] - (self.center[0]
-                            + sqrt(0.75)*self.pitch[0]*idx[0])
+                            + sqrt(0.75) * self.pitch[0] * idx[0])
             y = point[1] - (self.center[1]
-                            + (0.5*idx[0] + idx[1])*self.pitch[0])
+                            + (0.5 * idx[0] + idx[1]) * self.pitch[0])
```

Comment:
Same comments for above with respect to PEP8.

Reply:
I left a more detailed message on the prior comment. Since the readability benefit is only marginal, I went ahead and reverted the change.
******
<p align="left">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>